frameworks/helloworld/src/main/dist/svc.yml

```diff
@@ -9,11 +9,11 @@ pods:
     tasks:
       server:
         goal: RUNNING
-        cmd: "echo hello >> hello-container-path/output && sleep $SLEEP_DURATION"
+        cmd: "echo hello-$POD_INSTANCE_INDEX >> $OUTPUT_VOLUME/$OUTPUT_FILENAME && sleep $SLEEP_DURATION"
```

Comment:
The changes here in svc.yml are illustrative, but I think a little complex for the default hello-world example.  Do you have objections to moving this to one of the example yamls, or a new example yaml (with an integration test that actually exercises code)?